src/rsz/test/cpp_tests.ok

```diff
@@ -0,0 +1,19 @@
+Running test: TestBufRem1
+**********
+0
+Running main() from /home/cmoon/OR/OpenROAD/build/_deps/googletest-src/googletest/src/gtest_main.cc
```

Comment:
This ok file is machine and user dependent

Reply:
OK file has been removed.